pyhgvs/variants.py

```diff
@@ -173,13 +173,13 @@ def _on_forward_strand(self):
             seq_3p = self.seq_3p
             self.seq_5p = revcomp(seq_3p)
             self.seq_3p = revcomp(seq_5p)
-            self.alleles = map(revcomp, self.alleles)
+            self.alleles = list(map(revcomp, self.alleles))
```

Comment:
Nit: a list comprehension would be more idiomatic (and prevent duplicating memory overhead in python 2).